packages/pigeon/lib/swift_generator.dart

```diff
@@ -614,7 +621,7 @@ class SwiftGenerator extends StructuredGenerator<SwiftOptions> {
     });
   }
 
-  void _writeWrapError(Indent indent) {
+  void _writeWrapError(SwiftOptions generatorOptions, Indent indent) {
     indent.newln();
     indent.write('private func wrapError(_ error: Any) -> [Any?] ');
     indent.addScoped('{', '}', () {
```

Comment:
@stuartmorgan I'm of the general opinion that leaving the legacy option here is unnecessary. What are your thoughts?

Reply:
That would silently change behavior of existing code, which we try to avoid. Breaks that require people to update to compile are fine, but this won't do that, so I think we should leave it.

We should put it after the expected codepath though.